backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepositoryFactory.java

```diff
@@ -57,7 +57,7 @@ public GitMacheteRepositoryFactory() {
   @Override
   public IGitMacheteRepository create(Path mainDirectoryPath, Path gitDirectoryPath, IBranchLayout branchLayout)
       throws GitMacheteException {
-    LOG.startTimer().debug(() -> "Entering: mainDirectoryPath = ${mainDirectoryPath}, gitDirectoryPath = ${gitDirectoryPath}");
+    LOG.startTimer().info("Entering: mainDirectoryPath = ${mainDirectoryPath}, gitDirectoryPath = ${gitDirectoryPath}");
```

Comment:
Is this as important?

Reply:
Yeah, can be `debug` (with the `time elapsed` message still as `info`), will fix on another PR

Reply:
Done on PR #337